.env.example

```diff
@@ -15,7 +15,8 @@ FORWARD_PHPMYADMIN_PORT=64080
 
 # Feature Flags
 
-FEATURE_BEAT=true
+## example
+#FEATURE_BEAT=true
```

Comment:
There's also a reference in `.env.testing`

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/pull/2059/commits/e57d45732f1ccd3c1694bb9eb63b62d26222a0f6.

See also: https://github.com/RetroAchievements/RAWeb/pull/2059#discussion_r1405221043